packages/core/src/components/AutocompleteField/AutocompleteField.scss

```diff
@@ -0,0 +1,117 @@
+// Form-specific variables can be found in _variables.forms
+@import '@cmsgov/design-system-support/src/settings/index';
+
+/*
+Autocomplete field
+
+@paypal https://github.com/paypal/downshift#itemtostring
+
+Autocomplete fields allow users to enter any combination of letters, numbers, or symbols of their choosing (unless otherwise restricted), and receive one or more suggested matches in a list below the input.
+
+Markup:
+<div class="ds-u-clearfix ds-c-autocomplete">
+  <div class="ds-u-clearfix">
+    <label class="ds-c-label ds-u-margin-top--0 ds-c-autocomplete__label" for="autocomplete_7">
+      <span class="">What zip code did this person live during 2017?</span>
+      <span class="ds-c-field__hint">This is an autocomplete field. Begin typing to search for relevant information. The number of results will be updated as you type.</span>
+    </label>
+    <input aria-autocomplete="list" aria-expanded="false" autocomplete="off" class="ds-c-field ds-c-autocomplete__input" id="autocomplete_7" name="Downshift_autocomplete" role="combobox" type="text" value="">
+  </div>
+  <a aria-label="Clear typeahead and search again" class="ds-c-button ds-c-button--transparent ds-c-button--small ds-u-float--right ds-u-padding-right--0 ds-c-autocomplete__button" href="javascript:void(0);">Search again</a>
+</div>
+
+Style guide: components.autocomplete-field
+*/
+
+// autocomplete component
+.ds-c-autocomplete {
+  position: relative;
+  width: 460px;
+
+  &__inner--disabled {
+    + .ds-c-autocomplete__button {
+      background: transparent;
+      border: transparent;
+      color: $color-gray-dark;
+      pointer-events: none;
+    }
+  }
+  
+  &__label {
+    > .ds-c-field__hint {
+      border: 0;
+      clip: rect(0, 0, 0, 0);
+      height: 1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+      word-wrap: normal;
+    }
+  }
+
+  &__list {
+    background-color: $color-white;
+    max-width: 442px;
```

Comment:
I think this should use the `$input-max-width` value, but I'm not sure why it's smaller than the field's max-width?

But also, if we're setting its `width` to `100%`, then I don't think we need `max-width`, since the `max-width` on the parent `ds-c-autocomplete` would handle this for us.

Reply:
Done. Added a `box-sizing: border-box;` declaration to force the list container to honor the right width and not be wider than the input.